src/google/protobuf/compiler/objectivec/objectivec_helpers.h

```diff
@@ -46,6 +46,19 @@ namespace protobuf {
 namespace compiler {
 namespace objectivec {
 
+// Get/Set if the proto package should be used as to make the default prefix for
```

Comment:
not sure how to parse "used as to make the default"

Reply:
tweaked